src/OpenTelemetry.Instrumentation.StackExchangeRedis/TracerProviderBuilderExtensions.cs

```diff
@@ -26,6 +27,71 @@ namespace OpenTelemetry.Trace;
 /// </summary>
 public static class TracerProviderBuilderExtensions
 {
+    /// <summary>
+    /// Enables automatic data collection of outgoing requests to Redis.
+    /// </summary>
+    /// <remarks>
+    /// Note: A <see cref="IConnectionMultiplexer"/> will be resolved using the
+    /// application <see cref="IServiceProvider"/>.
+    /// </remarks>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static TracerProviderBuilder AddRedisInstrumentation(
```

Comment:
These overloads are a breaking change. An update from 9.8 to latest caused a more specific overload to be called which threw an argument null exception:

```b.AddRedisInstrumentation(null, c => {
                                c.EnrichActivityWithTimingEvents = false;
                                c.SetVerboseDatabaseStatements = config.FullDetails;
                            });```

Reply:
Sorry about that @niemyjski! It is a prerelease project so we reserve the right to make breaking changes until it goes stable. We try not to do that, but the overloads being added here are more or less the standard ones across the instrumentation from the core repo and much of contrib so it felt needed.

If you switch to...

```csharp
b.AddRedisInstrumentation(c => {
                                c.EnrichActivityWithTimingEvents = false;
                                c.SetVerboseDatabaseStatements = config.FullDetails;
                            });
```

...does that work for you?

Reply:
Yes, I found that fix and left the comment above. I just wanted to bring it to attention as it wasn't in the release notes as a breaking change. Thanks again!